### DISCUSSION TITLE
Pseudo 3d EBSD data

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#875] [Feature Requests]
Allow to work with EBSD sections at planes different than the xy plane. 

* allow for skew plots
* distinguish slices by storing the third dimension
* perform rotations / transformations in 3d

**Comment by Tijmenvermeij** [comment]
Hi Ralf,

I think it would be a very useful if Mtex has some 3D plotting/processing features. I understand there is some hesitation do do anything in 3D with Mtex, because there are other packages that already address this. However, we are now working with quasi-3D data in which 2D-EBSD scans can have "irregular" z values, because we collect EBSD slices for which we also measure the slice thickness profile. 
As far as I know, there's no software available that can handle this kind of data...

Tijmen

**Comment by ralfHielscher** [comment]
Most of the goals have accomplished with MTEX 6.0.

**Reply by SunilAnandatheertha** [reply]
So, would it be possible to use a stack of CTF files like we can do in Dream3D? All CTF files are perfectly aligned to the pixel.
